wsgi/odyssy/odyssy/settings.py

```diff
@@ -36,6 +36,7 @@
 DEBUG = not ON_OPENSHIFT
 
 ALLOWED_HOSTS = [
+    '127.0.0.1',
```

Comment:
`localhost` IP address is always `127.0.0.1`. If we allowed all hosts, it may pose a security threat when website will be live.